fe/fe-core/src/main/java/com/starrocks/mysql/privilege/Auth.java

```diff
@@ -827,6 +827,26 @@ public void replayGrant(PrivInfo privInfo) {
     private void grantInternal(UserIdentity userIdent, String role, TablePattern tblPattern,
                                PrivBitSet privs, boolean errOnNonExist, boolean isReplay)
             throws DdlException {
+
+        // check privilege
+        switch (tblPattern.getPrivLevel()) {
```

Comment:
If the isReplay is true, you should tolerate this error to make sure compatible.

Reply:
Good advice, I totally forget it.